components/salsa-macros/Cargo.toml

```diff
@@ -11,7 +11,9 @@ description = "Procedural macros for the salsa crate"
 proc-macro = true
 
 [dependencies]
+eyre = "0.6.5"
```

Comment:
we use this? should probably look into that...

Reply:
Filed https://github.com/salsa-rs/salsa/issues/504

Reply:
It's used in the `lazy-input` example, but not in the macro crate. I'll put up a PR.